lib/node/NodeWatchFileSystem.js

```diff
@@ -57,7 +57,8 @@ class NodeWatchFileSystem {
 				changes.filter(file => dirs.includes(file)).sort(),
 				changes.filter(file => missing.includes(file)).sort(),
```

Comment:
For performance reasons, you could convert `files`, `dirs`, `missing` into Sets.

And add a TODO to change the `watch(files, dirs, missing` signature to `Set`s in webpack 5.

Reply:
I'll change them into sets and will also take the TODO to change the signature in Webpack 5.